index.html

```diff
@@ -1,125 +1,161 @@
-            <li class="black"><p>Black<span>#000000</span></p></li>
-            <li class="magenta"><p>Magenta<span>#f70a7c</span></p></li>
-            <li class="green"><p>Green<span>#D9F1F1</span></p></li>
-            <li class="orange"><p>Orange<span>#ffcc00</span></p></li>
-            <li class="light-green"><p>Light Green<span>#D9F1F1</span></p></li>
-          </ul>
-          <h3>Secondary colors</h3>
-          <ul id="color-samples">
-            <li class="dark-grey"><p>Dark Grey<span>#272727</span></p></li>
-            <li class="light-grey"><p>Light Grey<span>#eff7f7</span></p></li>
-          </ul>
+        <div id="color" class="section">
+          <h1>Color</h1>
+          <div class="section-content">
+            <h2>Primary colors</h2>
+            <ul id="color-samples">
+              <li class="green"><p>Green<span>#00d242</span></p></li>
+              <li class="yellow"><p>yellow<span>#fdca31</span></p></li>
+              <li class="black"><p>Black<span>#222222</span></p></li>
+              <li class="red"><p>Red<span>#ff0000</span></p></li>
+            </ul>
+            <h2>Secondary colors</h2>
+            <ul id="color-samples">
+              <li class="dark-grey"><p>Grey Medium<span>#323235</span></p></li>
+              <li class="light-grey"><p>Grey Light<span>#dadada</span></p></li>
+              <li class="white"><p>White<span>#f8f8f8</span></p></li>
+            </ul>
+          </div>
         </div>
 
-        <div id="typography">
-          <h2>Typography</h2>
+        <div id="typography" class="section">
+          <h1>Typography</h1>
+          <div class="section-content">
+            
+            <ul class="font-family">
+              <li class="light">Open Source Light <span> Weight: 300</span></li>
+              <li class="medium">Open Source Medium <span> Weight: 500</span></li>
+              <li class="bold">Open Sans Bold <span> Weight: 800</span></li>
+            </ul>
 
-          <h3>Font Family</h3>
-          <ul class="font-family">
-            <li class="light">Open Sans Light <span> Weight: 300</span></li>
-            <li class="normal">Open Sans Regular <span> Weight: 400</span></li>
-            <li class="bold">Open Sans Bold <span> Weight: 700</span></li>
-          </ul>
-          <h3>Headings</h3>
-          <div id="headings">
-            <h1 class="h1">This is a large heading</h1>
-            <span>Weight 300, 60pt</span>
+          </div>
+          <div class="section-content">
 
-            <h2 class="h2">This is a medium heading</h2>
-            <span>Weight 300, 41pt</span>
+            <h1>H1, Weight 800, 2.625rem</h1>
+            <h2>H2, Weight 300, 1.375rem</h2>
+            <p>P, Weight 500, 1rem: FastRuby helps <code>small</code> businesses and entrepreneurs easily setup and <mark>run their online shops</mark>. We spend our days <strong>helping businesses sell more and improving</strong> eCommerce in Latin America.</p>
```

Comment:
@nykka Looks like the texts talk about OmbuShop and OmbuLabs, maybe it's cleaner to use Lorem Ipsum copy instead?

Reply:
@lubc I don't think Lorem Ipsum is a good idea. 

We want to use this space to hold copy about fastruby.io -- so please just take a sentence from fastruby.io and use it here instead of Lorem Ipsum. 

When it comes to Lorem Ipsum this is what I think: https://signalvnoise.com/archives2/getting_real_design_tip_just_say_no_to_lorem_ipsum.php

/cc @nykka